xds/server_test.go

```diff
@@ -656,8 +664,12 @@ func (s) TestHandleListenerUpdate_ErrorUpdate(t *testing.T) {
 	}
 }
 
+// TODO: Use serving state callback to get this test to work. Currently it
```

Comment:
Please remove this before submitting.  You can copy/paste into a gist/etc if there are edits here you need to keep for later.